.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: php
+
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
+php:
+  - 5.3
+  - 5.4
+  - 5.5
+  - 5.6
+  - 7.0
+
+install:
+  - composer install --prefer-source
+
+before_script:
+  - composer selfupdate
+  - composer update --prefer-dist
+
+script:
+  - phpunit
```

Comment:
default Travis script for php builds is `phpunit`([ref](https://docs.travis-ci.com/user/languages/php/#PHPUnit)), so imo could be removed